src/cascadia/TerminalSettingsEditor/ColorToHexConverter.cpp

```diff
@@ -0,0 +1,34 @@
+#include "pch.h"
+#include "ColorToHexConverter.h"
+#include "ColorToHexConverter.g.cpp"
+
+using namespace winrt::Windows;
+using namespace winrt::Windows::UI::Xaml;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    Foundation::IInspectable ColorToHexConverter::Convert(Foundation::IInspectable const& value,
+                                                          Windows::UI::Xaml::Interop::TypeName const& /* targetType */,
+                                                          Foundation::IInspectable const& /* parameter */,
+                                                          hstring const& /* language */)
+    {
+        til::color test{ winrt::unbox_value<winrt::Windows::UI::Color>(value) };

```

Comment:
The variable is named test?

Reply:
LOL that's me giving random var names while testing to see if this actually works then forgetting to change it once it works